array-methods/filter.js

```diff
@@ -0,0 +1,26 @@
+const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const names = [
+  'Ada',
+  'Hedy',
+  'Jean',
+  'Grace',
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna',
+];
+
+const evenNumbers = numbers.filter((Array) => Array % 2 === 0);
```

Comment:
I would use a different var name because a) "`Array`" is actually the current element being processed, b) Capital var names are used in React to demote a React element! (we will learn about this later) 

Suggestions: 
Okay - `element`, `el`, `x`
BEST - `number`, `num` (take advantage of array being processed's name!)

Reply:
This applies to the rest of your PR as well!